MyMusic/src/test/java/com/ciandt/summit/bootcamp2022/controller/GlobalExceptionHandlerTest.java

```diff
@@ -0,0 +1,35 @@
+package com.ciandt.summit.bootcamp2022.controller;
+
+import com.ciandt.summit.bootcamp2022.dto.ErrorDto;
+import com.ciandt.summit.bootcamp2022.exceptions.MinLengthRequiredException;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.junit.MockitoJUnitRunner;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+@SpringBootTest
+@RunWith(MockitoJUnitRunner.class)
+class GlobalExceptionHandlerTest {
+
+    @InjectMocks
+    private GlobalExceptionHandler globalExceptionHandler;
+
+    @Test
+    @DisplayName("Argument size is lass than three")
```

Comment:
corrigir o "lass" para "less". E acho que podemos colocar aqui no display o que esperamos também desse teste. Por exemplo: Argument size is less than three then return ....